phenotrex/cli/predict.py

```diff
@@ -10,6 +8,16 @@
               required=False, help='Input genotype file.')
 @click.option('--classifier', required=True, type=click.Path(exists=True),
               help='Path of pickled classifier file.')
+@click.option('--out_explain_per_sample', type=click.Path(dir_okay=False),
+              help='Write SHAP explanations for each predicted sample to file (optional).')
+@click.option('--out_explain_summary', type=click.Path(dir_okay=False),
+              help='Write SHAP explanations summarized over all samples. (Optional).')
```

Comment:
```suggestion
              help='Write SHAP explanations summarized over all samples (optional).')
```
for consistency with the line above